assets/css/screenplay.scss

```diff
@@ -59,3 +59,8 @@ body {
 .bootstrap-line-icon {
   stroke: white;
 }
+
+button,
+input[type="button"] {
+  letter-spacing: 0.5px;
+}
```

Comment:
All buttons with class `screenplay-button` are also of type `button`. So go ahead and remove `letter-spacing: 1px` from the above CSS, since one will override the other